.github/workflows/node.js.yml

```diff
@@ -0,0 +1,14 @@
+name: Node.js CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
```

Comment:
We should add a name to this job

Reply:
I thought this key was the job name. If you check the action run, it's listed as `build` on the sidebar, but you can change it to something else and that will appear.

Reply:
yes but we can still supply it with a more descriptive name by adding a [name](https://github.com/react-bootstrap/react-bootstrap/blob/f14ca04848bbfe52b6655e9bba5408ec87a53c8e/.github/workflows/ci.yml#L13) property to it (granted that link isn't a great example 😆)

Reply:
What would it be other than CI? That makes sense to me

Reply:
lol true. I'll go ahead and approve the PR either way since I don't want this to a blocker